src/lang/en-US.ts

```diff
@@ -479,10 +479,15 @@ const CaptureCreate: ResolvedIntlConfig['messages'] = {
     'captureCreate.editor.default': `Before you can edit the capture specification, you must fill out the Connection Configuration section and click "${CTAs['cta.generateCatalog.capture']}." `,
     'captureCreate.finalReview.instructions': `The following Flow specification was generated from the details you provided. To make changes, you can enter new values in the form above and click "${CTAs['cta.generateCatalog.capture']}" again. You can also edit the specification file directly. Click "${CTAs['cta.saveEntity']}" to proceed.`,
 
+    'captureCreate.collections.heading': `Output Collections`,
+    'captureCreate.collectionSelector.heading': `Collection Selector`,
+    'captureCreate.collectionSelector.instructions': `The collections bound to your existing capture. To make changes, you can enter new values in the this section of the form or edit the YAML file shown in the ${EntityCreate['entityCreate.catalogEditor.heading']} section below.`,
```

Comment:
The use of the word _existing_ makes this messaging flawed for the capture create workflow. At the very least, _existing_ should be dropped. Would that change suffice?

For reference, the materialization analog of this message is: _Choose one or more collections to materialize._